hashring/hashring.go

```diff
@@ -76,6 +80,7 @@ func New(hashfunc func([]byte) uint32, replicaPoints int) *HashRing {
 		hashfunc: func(str string) int {
 			return int(hashfunc([]byte(str)))
 		},
+		logger: logging.Logger("ring"),
```

Comment:
Ideally, the logger has the "local" field set to the node's hostport. In this particular case, it feels wrong to pass that info just to construct the logger, so I'm fine with the way it is.